schemas/sample_registration.json

```diff
@@ -0,0 +1,126 @@
+{
+  "name": "sample_registration",
+  "description": "Collection of elements required to register the required Donor-Specimen-Sample data to the ARGO Platform. Registration of samples is required before genomic and clinical data submission can proceed.",
+  "fields": [
+    {
+      "name": "program_id",
+      "valueType": "string",
+      "description": "Unique identifier of the ARGO program.",
+      "restrictions": {
+        "required": true
+      }
+    },
+    {
+      "name": "submitter_donor_id",
+      "valueType": "string",
+      "description": "Unique identifier of the donor, assigned by the data provider.",
+      "restrictions": {
+        "required": true,
+        "regex": "[A-Za-z0-9\\-\\._]{1,64}"
+      }
+    },
+    {
+      "name": "gender",
+      "valueType": "string",
+      "description": "Description of the donor self-reported gender. Gender is described as the assemblage of properties that distinguish people on the basis of their societal roles.",
+      "restrictions": {
+        "required": true,
+        "codeList": [
+          "Male",
+          "Female",
+          "Other"
+        ]
+      }
+    },
+    {
+      "name": "specimen_submitter_id",
+      "valueType": "string",
+      "description": "Unique identifier of the specimen, assigned by the data provider.",
+      "restrictions": {
+        "required": true,
+        "regex": "[A-Za-z0-9\\-\\._]{1,64}"
+      }
+    },
+    {
+      "name": "specimen_tissue_source",
+      "valueType": "string",
+      "description": "Tissue source of the biospecimen.",
+      "restrictions": {
+        "required": true,
+        "codeList": [
+          "Blood derived",
+          "Blood derived - bone marrow",
+          "Blood derived - peripheral blood",
+          "Bone marrow",
+          "Buccal cell",
+          "Lymph node",
+          "Solid tissue",
+          "Plasma",
+          "Serum",
+          "Urine",
+          "Cerebrospinal fluid",
+          "Sputum",
+          "NOS (Not otherwise specified)",
+          "Other",
```

Comment:
what's the difference between `NOS` and `Other`?

Reply:
@hknahal do you know the answer here? I only know that i have seen both...

Reply:
Let's take NOS out (but keep "Other").  NOS ("not otherwise specified") is more relevant for ICD codes. 


Although it's not related to this field, I just wanted to explain the difference anyways:
"NOS" ("not otherwise specified") is used when the medical report is insufficient to assign a more specific ICD code. The term "Other" (actually called "other specified" or "not elsewhere specified (NEC)") is used when the medical report provides detail for a code that 
does not exist in ICD, so the data submitter has to choose a more generalized code (ie. "as good as it gets"). There are actually ICD codes that contain the "NOC" or "NEC" abbreviation to denote these two types of codes.

Reply:
I've taken it out.  Thanks for the explanation!